compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp

```diff
@@ -581,6 +581,18 @@ bool QuantizeWithMinMaxPass::run(loco::Graph *g)
     circle_node->accept(&qb);
   }
 
+  for (auto node : loco::output_nodes(g))
```

Comment:
Can you please add a commnet what this `for` loop does as you explained in the comment?

Reply:
Something like this maybe...
1) update output nodes dtype
2) update output dtype

Reply:
I've added a comment.